assets/js/base/context/providers/cart-checkout/payment-methods/payment-method-data-context.tsx

```diff
@@ -222,7 +222,7 @@ export const PaymentMethodDataProvider = ( {
 	// When checkout is returned to idle, set payment status to pristine but only if payment status is already not finished.
```

Comment:
This comment also needs updated, just replace pristine with idle here `set payment status to pristine but only`. It might help to leave some comment with the reason why this is necessary